multimodal_transformers/model/tabular_transformers.py

```diff
@@ -563,7 +569,7 @@ def __init__(self, hf_model_config):
                                           hidden_channels=dims,
                                           bn=True)
 
-    @ add_start_docstrings_to_callable(XLM_INPUTS_DOCSTRING)
+    @ add_start_docstrings_to_model_forward(XLM_INPUTS_DOCSTRING)
     def forward(
```

Comment:
does it follow the standard output of huggingface models? 
reference: https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_outputs.py#L665-L699 

if no - let's add this as a future task